### PR TITLE
Added start date display in pre registration

### PR DESCRIPTION
**Pull request by Anunaya07**
## Proposed changes:
Added start date display if it is set by the user.

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Review by akshatnema — CHANGES_REQUESTED** [review]
Remove the changes of requirements.txt. They are of previous versions.

File: FusionIIIT/applications/academic_procedures/views.py
Comment (Aksh-Bansal-dev) [inline FusionIIIT/applications/academic_procedures/views.py]: Remove this print statement.

**Review by akshatnema — APPROVED** [review]
Approved :heavy_check_mark:. Waiting for the review from @Aksh-Bansal-dev.

**Review by Aksh-Bansal-dev — APPROVED** [review]

**Pull request merged**
